packages/api-client/src/conversation/Conversation.ts

```diff
@@ -107,3 +108,24 @@ export interface MLSConversation extends Conversation {
   cipher_suite: number;
   protocol: ConversationProtocol.MLS;
 }
+
+export interface MLS1to1Conversation {
+  conversation: MLSConversation;
+  public_keys?: {
+    removal: MLSPublicKeyRecord;
+  };
+}
+
+export function isMLS1to1Conversation(response: unknown): response is MLS1to1Conversation {
+  if (typeof response === 'object' && response !== null) {
+    const conversation = response as MLS1to1Conversation;
+
+    if (!!conversation.conversation && !!conversation.public_keys) {
+      return true;
+    }
+
+    return false;
+  }
+
+  return false;
```

Comment:
It is possible to normalize the logic?
Can we return early with false value?

Reply:
Something like this?

```typescript
export function isMLS1to1Conversation(response: unknown): response is MLS1to1Conversation {
  if (typeof response === 'object' && response !== null && 'conversation' in response && 'public_keys' in response) {
    return true;
  }

  return false;
}